cmd/build/helmify/static/README.md

```diff
@@ -81,9 +81,11 @@ _See [Exempting Namespaces](https://open-policy-agent.github.io/gatekeeper/websi
 | validatingWebhookTimeoutSeconds              | The timeout for the validating webhook in seconds                                      | `3`                                                                       |
 | validatingWebhookFailurePolicy               | The failurePolicy for the validating webhook                                           | `Ignore`                                                                  |
 | validatingWebhookCheckIgnoreFailurePolicy    | The failurePolicy for the check-ignore-label validating webhook                        | `Fail`                                                                    |
+| validatingWebhookExemptNamespacesLabels      | Additional namespace labels that will be exempt from the mutating webhook              | `{}`                                                                    |
```

Comment:
For this and the mutating webhook we probably want to also cite the security implications of adding more exemption labels. Namely that, unless there is a separate webhook controlling what labels can be applied to a namespace, the ability to create/modify namespaces is equivalent to the ability to bypass policy for resources in that namespace.

Reply:
nit: this should say `exempt from the validating webhook`